build_library/catalyst.sh

```diff
@@ -207,58 +198,50 @@ catalyst_init() {
     # so far so good, expand path to work with weird comparison code below
     FLAGS_seed_tarball=$(readlink -f "$FLAGS_seed_tarball")
 
-    if [[ ! "$FLAGS_seed_tarball" =~ .*\.tar\.bz2 ]]; then
-        die_notrace "Seed tarball doesn't end in .tar.bz2 :-/"
+    if [[ ! "$FLAGS_seed_tarball" =~ .\.tar\.(bz2|xz) ]]; then
+        die_notrace "Seed tarball doesn't end in .tar.bz2 or .tar.xz :-/"
```

Comment:
why - is the compression configurable? on bincache i see we're still producing a bz2 file.

Reply:
Gentoo switched to xz a while back, and this is left over from when I was trying a vanilla Gentoo seed. That turned out to not be viable, but I also used xz when making my own seed. It doesn't hurt. bzip2 only wins at compression speed anyway, which is why Gentoo switched.